src/frontend/package.json

```diff
@@ -8,17 +8,24 @@
     "start": "next start"
   },
   "dependencies": {
+    "@chakra-ui/core": "^0.8.0",
```

Comment:
Get rid of any dependencies you aren't actually using

Reply:
fixed